src/public/style.css

```diff
@@ -55,3 +55,70 @@ a {
 a:hover {
   color: #363636;
 }
+
+#errors {
```

Comment:
Let's remove all of this `#errors` styling. We won't need it until we do form validation, and by then we will be using a CSS framework that will provide these styles.